src/test/java/com/bunq/sdk/BunqSdkTestBase.java

```diff
@@ -4,51 +4,190 @@
+    OkHttpClient client = new OkHttpClient();
+
+    Request request = new Request.Builder()
+        .url(
+            SCHEME_HTTPS +
+            ApiEnvironmentType.SANDBOX.getBaseUri() +
+                URL_PATH_SEPARATOR +
+            ApiEnvironmentType.SANDBOX.getApiVersion() +
+            URL_PATH_SANDBOX_USER
+        )
+        .post(RequestBody.create(null, new byte[INDEX_FIRST]))
+        .addHeader(ApiClient.HEADER_REQUEST_ID, UUID.randomUUID().toString())
+        .addHeader(ApiClient.HEADER_CACHE_CONTROL, ApiClient.CACHE_CONTROL_NONE)
+        .addHeader(ApiClient.HEADER_GEOLOCATION, ApiClient.GEOLOCATION_ZERO)
+        .addHeader(ApiClient.HEADER_LANGUAGE, ApiClient.LANGUAGE_EN_US)
+        .addHeader(ApiClient.HEADER_REGION, ApiClient.REGION_NL_NL)
+        .build();
 
     try {
-      apiContext = ApiContext.restore(apiConfigPath);
-      apiContext.ensureSessionActive();
-    } catch (BunqException exception) {
-      List<String> ips = Arrays.asList(permittedIps);
-      apiContext = ApiContext.create(ApiEnvironmentType.SANDBOX, apiKey, DEVICE_DESCRIPTION, ips);
+      Response response = client.newCall(request).execute();
+      if (response.code() == HTTP_STATUS_OK) {
+        String responseString = response.body().string();
+        JsonObject jsonObject = new Gson().fromJson(responseString, JsonObject.class);
+        JsonObject apiKEy =jsonObject.getAsJsonArray(FIELD_RESPONSE)
+            .get(INDEX_FIRST)
+            .getAsJsonObject()
+            .get(FIELD_API_KEY)
+            .getAsJsonObject();
+
+        return SandboxUser.fromJsonReader(new JsonReader(new StringReader(apiKEy.toString())));
+      } else {
+        throw new BunqException(String.format(ERROR_COULD_NOT_GENERATE_NEW_API_KEY, response.body().string()));
+      }
+    } catch (IOException e) {
+      throw new BunqException(e.getMessage());
     }
+  }
+
+  private static void setSecondMonetaryAccountBank() {
+    BunqResponse<Integer> response = MonetaryAccountBank.create(CURRENCY_EUR, ACCOUNT_DESCRIPTION);
 
-    apiContext.save(apiConfigPath);
+    secondMonetaryAccountBank = MonetaryAccountBank.get(response.getValue()).getValue();
+  }
+
+  private static void requestSpendingMoney() {
+    if (shouldMoneyBeRequested(BunqContext.getUserContext().getPrimaryMonetaryAccountBank())) {
+      RequestInquiry.create(
+          new Amount(SPENDING_MONEY_AMOUNT, CURRENCY_EUR),
+          new Pointer(POINTER_TYPE_EMAIL, SUGAR_DADDY_EMAIL),
+          SUGAR_DADDY_REQUESTS_DESCRIPTION,
+          false
+      );
+    }
 
-    return apiContext;
+    if (shouldMoneyBeRequested(secondMonetaryAccountBank)) {
+      RequestInquiry.create(
+          new Amount(SPENDING_MONEY_AMOUNT, CURRENCY_EUR),
+          new Pointer(POINTER_TYPE_EMAIL, SUGAR_DADDY_EMAIL),
+          SUGAR_DADDY_REQUESTS_DESCRIPTION,
+          false,
+          secondMonetaryAccountBank.getId()
+      );
+    }
   }
 
-} 
+  protected static Pointer getPointerBravo() {
```

Comment:
And wasn't the convention to always add (empty) docblocks?

Reply:
i thought we agreed to there is no need for it anymore 🤔 as it didn't bring any value as its a strict typed lang.

Reply:
We agreed on being consistent I think. I have posted multiple times that they where not there and you added it multiple times as that was the way you did it before. Switching styles all the time is messy, do it or don't but make sure to do it consistent.

Reply:
Alrighty then, as its a strict lang, no need for empty doc blocks. in the non strict ones will add doc blocks to define parameters etc.

Reply:
Ok so you're gonna change this in the generator?

Reply:
yea, but in a future PR then. That might cost some time. Will create followup issue.

Reply:
https://github.com/bunq/sdk_java/issues/88